src/components/FullNameInputRow.js

```diff
@@ -48,6 +48,7 @@ const FullNameInputRow = (props) => {
             <View style={styles.flex1}>
                 <TextInput
                     label={props.translate('common.firstName')}
+                    name="fname"
```

Comment:
do we use this `name` prop at all? I don't see it come up in the `BaseTextInput` component

Reply:
I think this is what the browser uses to autofill the input.

Reply:
Ah, found where it's being used but still don't totally understand what it's for https://github.com/Expensify/App/blob/1ab42aba7527e201fda9784b791d371fdc479e98/src/components/TextInput/index.js#L13-L16

Maybe some follow-up documentation would be helpful @neil-marcellini

Reply:
Yeah as Carlos said it's what the browser uses to autofill the input. Here is the reference I followed https://developers.google.com/web/fundamentals/design-and-ux/input/forms#recommended_input_name_and_autocomplete_attribute_values
Sorry that I didn't see this yesterday.